### ISSUE TITLE
Swap out extensions for upstreams

### ISSUE THREAD

**Issue by CJ-Wright** [regro/cf-scripts#766]
Sometimes upstream changes the extension. It might be nice if the bot tried a few things (similar to the `v` stuff) to get a viable upstream.

**Comment by beckermr** [comment]
I think we should not automerge things like this

**Comment by beckermr** [comment]
Like the only thing that gets auto merged is a straight version bump

**Comment by CJ-Wright** [comment]
Sure, that may make this a bit more complex (how do we let the main migrator know that we swapped out the extension). Also we should have a list of extensions in order of preference (if the standard one doesn't work)

**Comment by beckermr** [comment]
This turns out to be very expensive so not done right now.

**Comment by CJ-Wright** [comment]
Yeah I think we need to put that hashing stuff onto threads, we're most likely waiting on server communication.

**Comment by beckermr** [comment]
Right. It was in a separate process in dask but I am having a ton of trouble with the pr.

**Comment by beckermr** [comment]
I think actually we should move the pr updating and the version updating to their own processes in separate repos. This will put us in the road to dynamodb while giving us more breathing room.
